lib/praxis/tasks/api_docs.rb

```diff
@@ -9,20 +9,21 @@
         raise Exception.new("NPM Install Failed")
       end
 
-      docs_dir=File.join(Dir.pwd, 'docs')
+      docs_dir = File.join(Dir.pwd, 'docs')
       FileUtils.mkdir_p docs_dir unless File.directory? docs_dir
 
       # The doc browser will need to have a minimal app.js and styles.css file at the root
-      # Let's add them if the app has not overriden them 
-      js_file = File.join(Dir.pwd, 'docs', 'app.js') 
-      scss_file = File.join(Dir.pwd, 'docs', 'styles.scss')   
+      # Let's add them if the app has not overriden them
+      js_file = File.join(Dir.pwd, 'docs', 'app.js')
+      scss_file = File.join(Dir.pwd, 'docs', 'styles.scss')
+      template_directory = File.expand_path(File.join(File.dirname(__FILE__), '../../../tasks/thor/templates/generator/empty_app/docs'))
+
       unless File.exists? js_file
-        File.open(js_file, 'w') {|f| f.write(%q{angular.module('DocBrowser', ['PraxisDocBrowser']);}) }
```

Comment:
@gampleman , why would you rather depend on a thor generator file, than explicitly embed it here like I had?
Any changes we might want to apply to the empty app, should not break this no?
I don't feel too strongly, so I'll go either way, but just wanted to make sure you want to do it this way...since it seems much more brittle.